bitrise.yml

```diff
@@ -426,6 +426,18 @@ workflows:
     - swiftlint-extended@1:
         inputs:
         - linting_path: "$BITRISE_SOURCE_DIR"
+    - deploy-to-bitrise-io@1.9: {}
+    - script@1:
+        inputs:
+        - content: |-
+            set -e
+            echo "$SWIFTLINT_REPORT"
+            if [ ! -z "$SWIFTLINT_REPORT" ] ;then
```

Comment:
With this script we check if there is a swiftlint error, if that's the case the build fails. When there are no failures the report file is not created.